conf/pipeline/sarek/uppmax.config

```diff
@@ -0,0 +1,17 @@
+// sarek/uppmax specific profile config
+
+params {
+  config_profile_contact = 'Maxime Garcia (@MaxUlysse)'
+  config_profile_description = 'nf-core/sarek uppmax profile provided by nf-core/configs'
+}
+
+if (hostname ==~ "i.*") {
+  singularity {
+      cacheDir = params.containers_path
```

Comment:
I think we said that Irma-specific configs we might as well put in the irma-provision repositories. The users for that cluster is limited to NGI anyway.

Reply:
I think the current idea is to get everything out of irma provision